Client/src/Remote/Endpoints/PermissionsEndpoint.tsx

```diff
@@ -0,0 +1,24 @@
+import SnackbarUtils from '../../Components/Utils/SnackbarUtils'
+import { IUpdatePermissionInfo } from '../../Models/Permissions/IUpdatePermissionInfo'
+import { get, put } from "../FluentRequest"
+
+const updatePermissionsRoute = '/api/v1/manageAdmin'
+const getAdminsAllRoute = '/api/v1/manageAdmin'
+
+export const updatePermissions = async (updatePermissions: IUpdatePermissionInfo): Promise<any> => {
+    const result = await put(updatePermissionsRoute).withBody(updatePermissions).json()
+    if (result == 'Admin permissions successfully revoked') {
+        SnackbarUtils.success(result)
+    }
+    else if (result == 'User successfully given admin permissions') {
```

Comment:
the fluent request has a call() function that returns the Response object, where you can see the success status code, its a bad idea to check on the returned message + in both cases you display success(result), and the error is already handled inside the FluentRequest, what im saying is just check if returned status is 2xx and display success and thats it